pkg/metrics/reconciliation_status.go

```diff
@@ -53,7 +53,7 @@ func (c *ReconciliationStatusCollector) OnClusterStateUpdate(state *cluster.Stat
 	}
 
 	c.reconciliationStatusGauge.
-		WithLabelValues(state.Cluster.RuntimeID, state.Cluster.Runtime.Name).
+		WithLabelValues(state.Cluster.RuntimeID, state.Cluster.Runtime.Name, state.Cluster.Metadata.GlobalAccountID, state.Cluster.Metadata.ServicePlanID).
```

Comment:
Shouldn't we use plan name instead of plan ID as it's much more user friendly?

Reply:
Sounds great and I'll change to `state.Cluster.Metadata.ServicePlanName`.